libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/PropertySchema.cs

```diff
@@ -116,7 +108,13 @@ public PropertySchema(string path, JObject schema, List<PropertySchema> children
         /// Gets mappings to possible entities for property.
         /// </summary>
         /// <value>List of entity names.</value>
-        public IReadOnlyList<string> Mappings { get; }
+        public IReadOnlyList<string> Entities { get; }
```

Comment:
do these get serialized?  I see there is a JsonIgnore attribute, in which case, all of these properties should have JsonProperty attribtes with camelcase names on them.

Reply:
Does not get serialized.